src/features/joinForms/components/JoinFormListItem.tsx

```diff
@@ -86,7 +86,10 @@ const JoinFormListItem = ({ form, onClick }: Props) => {
         <FormatListBulleted className={classes.icon} />
         <Box>
           <Typography color={oldTheme.palette.text.primary}>
-            {form.title}
+            {form.title}{' '}
+            <Typography color={oldTheme.palette.text.disabled} component="span">
+              (#{form.id})
+            </Typography>
```

Comment:
This is just details, but usually when we have two separate texts that we display next to each other, we place them next to each other in th box with `alignItems='center'` and give a bit of `gap` or `margin` between them. 

Also, involving you in a bit of cleanup here, but I think it should use the `secondary` color instead of `disabled`.  This can actually be achived more simple than has been done in these lines of code - just `color='secondary'` will work great, and there is no need to specify the `color` property on the first `Typography` where it should be the primary color, that's just the default and does not need to be explicitly stated.

Reply:
Thanks for the feedback, these are good points! Will update with these.

Reply:
:D i know, it takes time to get familiar with the style and ways of a whole project - i'm still learning every time i make a PR after 3 years 😃